tests/test_pipelines_sentiment_analysis.py

```diff
@@ -1,12 +1,90 @@
 import unittest
 
+import numpy as np
+
+from transformers import AutoTokenizer, DistilBertConfig, DistilBertForSequenceClassification, pipeline
+from transformers.testing_utils import slow
+
 from .test_pipelines_common import MonoInputPipelineCommonMixin
 
 
+VALID_INPUTS = ["I really disagree with what you've said.", ["I love you."]]
+
+
 class SentimentAnalysisPipelineTests(MonoInputPipelineCommonMixin, unittest.TestCase):
     pipeline_task = "sentiment-analysis"
-    small_models = [
-        "sshleifer/tiny-distilbert-base-uncased-finetuned-sst-2-english"
-    ]  # Default model - Models tested without the @slow decorator
+    small_models = ["distilbert-base-cased"]  # Default model - Models tested without the @slow decorator
```

Comment:
This change was necessary as the previous model was not outputting valid results (`0.0` on everything).